docs/Single Sign On/okta.md

```diff
@@ -17,21 +18,57 @@ Login to your Okta account as an admin and navigate to `Applications -> Applicat
 
 In the dialog that opens, choose **OIDC - Open ID Connect** as the **Sign-in Method** and **Web Application** as the **Application Type.**
 
-On the next screen provide a name for the application. Under **Assignments -> Controlled Access**, select `Skip Group Assignment For Now`
-
 <img src="/img/docs/oidc-okta-1.png" alt="Creating New Okta OAuth App" height="75%" width="75%"/>
 
 Leave the **Application Callback URL** empty for now and Register the application.
 
-Note down the `client-id` and `client-secret` as those will be required to configure identity provider for Paralus.
+Based on who can access the Paralus app integration, select **Controlled access** options.
+
+Note down the `Client ID` and `Client Secret` as those will be required to configure identity provider for Paralus.
+
+### Add a custom claim for groups mapping
+
+Paralus comes with the in-built functionality for mapping Identity
+provider (IdP) groups to Paralus. This is useful to manage different
+Paralus access to the IdP users through IdP groups they are part
+of. It eases the access-management process for Parlaus
+administrators. See the limitation below.
+
+In order to achieve successful groups mapping, both Okta group and
+Paralus group must match their names. Once it matches, then IdP users
+can access the Paralus resources based on permissions assigned to
+matching Paralus group. Read how to manage Okta users and groups in
+[Okta
+documentation](https://help.okta.com/oie/en-us/Content/Topics/users-groups-profiles/usgp-groups-main.htm).
+
+Okta by default does not return groups information in OIdC
```

Comment:
```suggestion
Okta by default does not return groups information in OIDC
```

Also, can we link to the Okta document that mentions the scopes so the reader can see that groups are not returned? I found this link: https://auth0.com/docs/get-started/apis/scopes/openid-connect-scopes

Reply:
This is generic document about OIDC scopes and standard claims. I am not able to find a document from Okta.